### DISCUSSION TITLE
deploying react web with machine learning model

### DISCUSSION THREAD

**Discussion by shivShakya** [vercel/community#2685] [Help] [answered]
### Summary

Hi there ,
 How can i deploy a website made using react js and flask api but with a machine learning model involved. react is deployed easily but with machine learning it is being difficult. please help .
thank you

### Steps to Reproduce

.

**Comment by leerob** [comment, marked as answer]
Hey there! We have a template that uses Next.js on the frontend and Flask on the backend here: https://vercel.com/templates/next.js/nextjs-flask-starter
